### PR TITLE
Feature/homework3/hierarchy of classes

### PR DESCRIPTION
**Pull request by yalenna**

### CODE REVIEW

File: src/Homework3/Book.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import java.util.*;
+public class Book {
+    String ISBN;
+    String Name;
+    List<Author> Authors;
+    String Publisher;
+    String PublicationDate;
+    boolean Paperback;
+
+    Book() {
```
Comment (knadonenkoepam) [inline src/Homework3/Book.java:12]: Why this constructor is needed?

File: src/Homework2/Fibonacci.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package Homework2;
+
+public class Fibonacci {
```
Comment (knadonenkoepam) [inline src/Homework2/Fibonacci.java:3]: Files from homework2 is not needed in pullrequest for homework3

File: src/Homework3/Publisher.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package Homework3;
+
+public class Publisher {
+    int ID_pub;
+    String PubName;
+    String PubAddress;
+
+    Publisher() {
```
Comment (knadonenkoepam) [inline src/Homework3/Publisher.java:8]: Another empty constructor :) Why?

File: src/Homework3/Book.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
+package Homework3;
+
+import java.util.*;
```
Comment (knadonenkoepam) [inline src/Homework3/Book.java:3]: Wildcard imports are bad :)

**Review by knadonenkoepam — COMMENTED** [review]
Overal good work, got some minor comments. Also keep looking for code formating, IntelliJ provides autoformat function, you could use it with hotkey ctrl+alt+L for Windows or command + ⌥ + L on MacOS.
Also there should be no Homework2 in this pullrequest

File: src/Homework3/Book.java (hunk 1 above)
Comment (yalenna) [reply src/Homework3/Book.java]: It was added "just in case", deleted it.

File: src/Homework3/Publisher.java (hunk 3 above)
Comment (yalenna) [reply src/Homework3/Publisher.java]: It was added "just in case", deleted it.

File: src/Homework3/Book.java (hunk 4 above)
Comment (yalenna) [reply src/Homework3/Book.java:3]: Changed it to **import java.util.List**
And also in src/Homework3/Task.java to 
**import java.util.Arrays;
import java.util.List;**

**Review by knadonenkoepam — APPROVED** [review]

**Pull request merged**